ports/atmel-samd/boards/openbook_m4/pins.c

```diff
@@ -0,0 +1,71 @@
+#include "shared-bindings/board/__init__.h"
+
+#include "boards/board.h"
+#include "shared-module/displayio/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
+    { MP_OBJ_NEW_QSTR(MP_QSTR_A0),  MP_ROM_PTR(&pin_PA02) }, // A0 = audio right channel
+    { MP_OBJ_NEW_QSTR(MP_QSTR_A1),  MP_ROM_PTR(&pin_PA05) }, // A1 = audio left channel
+    { MP_OBJ_NEW_QSTR(MP_QSTR_A2),  MP_ROM_PTR(&pin_PB08) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_A3),  MP_ROM_PTR(&pin_PB09) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_A4),  MP_ROM_PTR(&pin_PA04) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_A5),  MP_ROM_PTR(&pin_PA06) },
+
+    { MP_OBJ_NEW_QSTR(MP_QSTR_A6),  MP_ROM_PTR(&pin_PB01) }, // A6 = VBAT Monitor
```

Comment:
We use `board.VOLTAGE_MONITOR` for other boards that have this kind of thing. Add more descriptive pin name aliases if you would like. I'll approve the PR but will not merge in case you want to add any more pin aliases. Let us know.

Reply:
On second thought, never mind for now. I'll merge in and you can submit another PR later.